switch.py

```diff
@@ -48,7 +42,7 @@ def enable():
     global overlay_process
     overlay_process = subprocess.Popen(
         [
-            f"{virtualenv_dir}/bin/python",
+            sys.executable,
             config.get_active_overlay(),
             "--host",
             brokerIP,
```

Comment:
The linter seems to complain about this line, it wants the entire array on one line